### ISSUE TITLE
Feature: Replace LLM Provider Type Box with Dropdown List

### ISSUE THREAD

**Issue by ashmilhussain** [sirocco-ventures/raggenie#18]
Change the LLM provider input from a type box to a dropdown list. This will allow users to select from a predefined list of available LLM providers, improving the user experience and reducing input errors.

**Comment by uchiha-vivek** [comment]
Sir please assign me this issue .

**Comment by ashmilhussain** [comment]
Hey @uchiha-vivek ,

Assigning this issue to you, happy coding 


If any assistance required meet our team here : https://join.slack.com/t/theailounge/shared_invite/zt-2ogkrruyf-FPOHuPr5hdqXl34bDWjHjw

**Comment by uchiha-vivek** [comment]
![Screenshot (91)](https://github.com/user-attachments/assets/c130a98a-8726-4383-99be-c5f4b193f27e)

The llm Provider input has drop down with three llm provider already  so do i need to add more of them ? @ashmilhussain

**Comment by ashmilhussain** [comment]
The idea is to make Endpoint and Providers also a drop down list

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek  any update on this ?

**Comment by uchiha-vivek** [comment]
@ashmilhussain yes sir working on it

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek any update on this ?

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k yes sir currently I am working actively on this issue, I will send y the updates of my initial work.
